.github/workflows/test-rules.yaml

```
@@ -11,7 +11,45 @@ on:
   workflow_call:
 
 jobs:
+  extract-languages:
+    runs-on: ubuntu-latest
+    outputs:
+      languages: ${{ steps.extract.outputs.languages }}
+    steps:
+      - uses: actions/checkout@v4
+      - id: extract
+        name: Extract languages from file
+        run: |
+          relative_path="crates/cli/src/datadog_utils.rs"
+          
+          if [ ! -f "$relative_path" ]; then
+            echo "Error: could not find target file"
```

We could use the native error syntax to log errors, which makes them more easily visible in the workflow summary (see [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-error-message)). I also think we could log the file (in case we move it around in the future), because we'd know immediately what the path is in the workflow summary. This removes the need to diginto the workflow itself if this fails.

```suggestion
            echo "::error::Could not find file `$relative_path`"
```